### PR TITLE
#1: added sample of VALUES array

### PR DESCRIPTION
**Pull request by tmtanzeel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,25 @@ Property	                  |	Type   	     |	Default	                      |	Desc
 This is how it can be used.
 
 ```
+import HorizontalTimeline from 'react-horizontal-timeline';
+
 const VALUES = [ /* The date strings go here */ ];
```
Comment (sherubthakur) [inline README.md:41]: Let's remove this!

File: README.md [hunk 2]
```diff
@@ -36,8 +36,25 @@ Property	                  |	Type   	     |	Default	                      |	Desc
...
+For example:
+const VALUES = [
+    '2008-06-01',
+    '2010-06-01',
+    '2013-06-01',
+    '2015-03-01',
+    '2019-01-01',
+    '2019-06-17',
+    '2019-08-01',
+];
```
Comment (sherubthakur) [inline README.md:54]: No need for this section to be a comment and this can be part of the showcase code.

**Review by nixypanda — COMMENTED** [review]
Overall I like this idea. Just mentioned a few changes, once those are done we can totally merge this.

File: README.md (hunk 1 above)
Comment (tmtanzeel) [reply README.md:41]: this import is required to fix the compiler error:
`Failed to compile.

src/components/EducationTimeline/EducationTimeline.js
  Line 40:22:  'HorizontalTimeline' is not defined  react/jsx-no-undef

Search for the keywords to learn more about each error.
`

Comment (sherubthakur) [reply README.md:41]: I was talking about removal of line `const VALUES = [ /* The date strings go here */ ];` and not the `import` one.

File: README.md (hunk 2 above)
Comment (tmtanzeel) [reply README.md:53]: addressed

**Comment by tmtanzeel** [comment]
Subject: Addressed review comments
Please revisit the PR (
https://github.com/sherubthakur/react-horizontal-timeline/pull/94)

Thanks and regards
Tanzeel Mirza
Visit website <http://www.tanzeel.online>

File: README.md (hunk 1 above)
Comment (tmtanzeel) [reply README.md]: removed (my apologies for the confusion)

**Comment by tmtanzeel** [comment]
Subject: Addressed all review comments

Thanks and regards
Tanzeel Mirza
Visit website <http://www.tanzeel.online>

**Comment by nixypanda** [comment]
Looks good! Thanks for the PR :)

**Pull request merged**